responses/it/HassTurnOff.yaml

```diff
@@ -2,8 +2,8 @@ language: it
 responses:
   intents:
     HassTurnOff:
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.name }}
-      fans_area: Turned off fans in {{ slots.name }}
-      cover: Closed {{ slots.name }}
-      cover_area: Closed {{ slots.area }}
+      default: spento {{ slots.name }}
```

Comment:
this way to handle is different from the others.
I personally prefer to align this one to the others and make something like:
"Ho spento {{ slots.name }}"

Reply:
if you don't mind, I am going to merge this PR now so some responses are in place. We can tweak the content afterwards.

Reply:
ok